example.env

```diff
@@ -111,3 +111,6 @@ SCHEDULE_NOTIFY_AND_DELETE_INACTIVE_USERS=9:00 pm
 
 # Time interval (in seconds) for polling submission report for automated tests.
 SUBMISSION_REPORT_POLL_TIME=30
+
+# Time(in minutes) for allowing coaches to review submissions of inactive students, keeping it 0 allow coaches to indefinitely review submissions, default to two weeks
```

Comment:
_default to two weeks_ doesn't seem right as the `secrets.yml` has it set to 30 mins.